docs/source/concept_models/vivarium_swissre_cervicalcancer/concept_model.rst

```diff
@@ -536,28 +545,31 @@ Vaccine coverage
    and alternative scenarios.
 
 Vaccine efficacy
- - Relative risk of getting HPV 16/18 infection for those vaccinated versus 
-   unvaccinated (RR_vaccine): use normal distribution **normal(mean=0.22, SD=0.04)**
+ - Zhu et al. reported a relative risk of getting HPV 16/18 infection for those 
+   unvaccinated versus vaccinated: use normal distribution 
+   **normal(mean=4.71, SD=0.94)**
```

Comment:
Is this ratio the way the software engineers prefer it?  It is the reciprocal of what you had before

Reply:
Yes, they prefer to use inverse distribution of RR for protection effect in order to make it always greater than 1.